### ISSUE TITLE
Logging

### ISSUE THREAD

**Issue by fwoelffel** [kuitos/axios-extensions#6]
Hi there

Currently, the only way to remove the log output from this module is to set `NODE_ENV` to production. 
Moreover, `axios` is expected to run in Node.js and in browsers: I don't think there is a way to set such a variable in the later environment.

Could we consider using the `debug` package instead of console.log ?
(See https://github.com/visionmedia/debug#browser-support)

**Comment by kuitos** [comment]
Maybe u need webpack to bundle your scripts and then do the configuration:

```js
new webpack.DefinePlugin({
  'process.env.NODE_ENV': JSON.stringify('production')
})
```
https://webpack.js.org/guides/production/#specify-the-environment

**Comment by fwoelffel** [comment]
Still, the logging is currently polluting my dev environment logs. I'm willing to submit a PR if you're interested in logging with `debug`.
This article may help you understand my concern https://blog.risingstack.com/node-js-logging-tutorial/
> It is not recommended to pollute the log files with your events when others use the module you build. However, in some cases, application developers may need it to understand better why a given issue arose.

**Comment by kuitos** [comment]
It designed for developer to help them realized which request had been cached, given them more runtime information, in my purpose.

The `debug` is awesome but import a non-standard logger module for that simple requirement is not a good idea, which will also increases integration difficulties with a nodejs manager log system, such as `pm2`.

Maybe I need to use another environment variable for logging switcher rather than `NODE_ENV`?
such as:
```js
new webpack.DefinePlugin({
  'process.env.LOGGER_LEVEL': JSON.stringify('error')
})
```
Then u can switch off logging in dev environment if u need.

**Comment by fwoelffel** [comment]
That'd be great :)

**Comment by kuitos** [comment]
released at v1.2.0, thanks for your feedback!😀
